lib/monitoring/librato_notifier.rb

```diff
@@ -11,9 +10,10 @@ def initialize(prefix: nil, logger: Rails.logger, type: :count, unit: "")
     #
     # results:: a hash where the keys represent the source (the resque instance name) and the values
     #           are lists of items to be counted.  The items won't be examined, just counted and used in the metric
-    def notify!(results)
-      results.each do |resque_name,items|
-        log_to_librato(resque_name,@type,@prefix,items.size)
+    def notify!(check_results)
+      check_results.each do |check_result|
+        source = [check_result.resque_name,check_result.scope].compact.join(".")
```

Comment:
This is how the `scope` is used.  Bascially if we think of the resque name as the source, the scope is a refinement inside that resque of what the source is.  In the case of the queue size check, it uses the name of the queue as the scope (so, in theory, we could alert on different queue sizes differently, which is what i intend to do with the failed job classes)